src/webhooks/webhooks.ts

```diff
@@ -0,0 +1,96 @@
+import { Webhook } from './interfaces/webhook.interface';
+import crypto from 'crypto';
+import { SignatureVerificationException } from '../common/exceptions';
+
+export class Webhooks {
+  constructEvent(
+    payload: any,
```

Comment:
Can we use options objects for these methods since they have a bunch of parameters?

```ts
constructEvent({
  payload,
  sigHeader,
  secret,
  tolerance = 180
}: {
  payload: any,
  sigHeader: string,
  secret: string,
  tolerance?: number
}): Webhook {

}
```